hooks/environment.d/999_cryptic_unlock.sh

```diff
@@ -9,7 +9,7 @@ set -eou pipefail
 ## agent should exit, causing the docker container to restart and restore the deleted files.
 ## This gives us the capability to deny access to these files to later steps within the
 ## current buildkite job.
-SECRETS_MOUNT_POINT="${BUILDKITE_PLUIGIN_CRYPYTIC_SECRETS_MOUNT_POINT:-/secrets}"
+SECRETS_MOUNT_POINT="${BUILDKITE_PLUIGIN_CRYPTIC_SECRETS_MOUNT_POINT:-/secrets}"
```

Comment:
Should this have been 
```suggestion
SECRETS_MOUNT_POINT="${BUILDKITE_PLUGIN_CRYPTIC_SECRETS_MOUNT_POINT:-/secrets}"
```
?

Reply:
Oh there were two typos, https://github.com/JuliaCI/sandboxed-buildkite-agent/pull/106